src/subtitling/Group setups/github.md

```diff
@@ -0,0 +1,145 @@
+You can create a repository by pressing the + button on the top-right of the page and clicking "New repository".
+
+![Create a new repository](../images/gh_make_repo1.png)
+
+You can then create a new repository under your own username or an organization you have writing access to.
+
+![Setting an owner and repository name](../images/gh_make_repo2.png)
+
+Give your repository a name,
+and decide to make your repo either public or private.
+You may also want to add a README if you want to share additional information about the project.
+You can then create the repository by pressing "Create repository".
+
+If you're working with multiple users,
+you'll want to make sure you to add them to the repository as well.
+To do this you go to your repository's settings,
+and then click on "Manage access".
+Here you can add users or teams (in an organization) to give access to your repository.
+You'll want to give them either "write" or "maintain" roles
+so they'll be able to properly make changes without many headaches.
+
+![Managing access](../images/gh_manage_access.png)
+
+
+[GitHub_home]: (https://github.com/)
+[GitLab_home]: (https://about.gitlab.com/)
+[BitBucket_home]: (https://bitbucket.org/)
+
+## Basic actions
+
+**If you are new to GitHub,
+we highly recommend you use the [GitHub desktop client](GitHub_desktop)**
+(this also works with other,
+similar platforms).
+This guide was written under the assumption that most users will be using this client.
+
+### Cloning
+
+After installing the desktop client,
+open it and log in.
+You can then clone the repository to your system by going to `File > Clone repository`,
+or by going to the repository on GitHub and going `Code > Open with GitHub Desktop`.
+
+![Opening a repo with the desktop client](../images/gh_clone.png)
+
+If the repository has not been set up yet,
+you can set it up in your desktop client through the link on the repository.
+This will automatically open up the cloning menu in your desktop client.
+
+![Setting up a repo with the desktop client](../images/gh_setup_repo.png)
+
+### Syncing file changes
+
+When editing files from a Git instance,
+Git will remember and verify what files have been changed.
+Before making any commits however,
+you'll want to first double-check that you have every file pulled to your local repository.
+Click "Fetch origin" at the top,
+and pull in any newly-pushed commits made by other users.
+
+Now you can verify your own changes and commit the files you want to upload/update in the repository.
+The desktop client will additionally give you a very useful display of what is different between your file
+and the latest version of the file you pulled in from the repo.
+To make things easier for your teammates,
+you'll want to make sure to give your commit a helpful title and description if necessary.
+Finally,
+press "Commit to **master**",
+and once you've finished making all your changes and committed them,
+press "Push origin".
```

Comment:
How does Github Desktop behave if there have been diverging changes on the remote? This is likely to occur and we should mention this, likely also refering to the merge conflict resolving section below.

Reply:
It will make a "merge commit". Haven't run into issues with it so far at all besides it creating another commit which can be annoying. If you're using the CLI to handle these issues though, I think it's safe to assume you know how to resolve it or google a way to resolve it.
![image](https://user-images.githubusercontent.com/42927908/121772862-7dadfa80-cb78-11eb-9315-e382ffffadb1.png)